File: app/src/main/java/com/artmurka/artmurkaapp/android/views/fragments/interfaces/ICheckoutFragment.java
package com.artmurka.artmurkaapp.android.views.fragments.interfaces;


import com.artmurka.artmurkaapp.model.pojo.itemlist.checkout.OrderDesc;
import com.artmurka.artmurkaapp.model.pojo.itemlist.novaposhta.Areas.AreasResponse;

import java.util.ArrayList;
import java.util.List;

public interface ICheckoutFragment {
    void showCheckout(List<OrderDesc> list);
    void refreshSumPrice(String price);
    void showOrderIsProcessed(String msg);
    void setDataSpinner(ArrayList<String> delivery, ArrayList<String> payment);
    void setSityes(String[] sityes);
    void showMessage(String message);
    void setAreas(AreasResponse areas);
    void setCities(List<String> cities);
    void setWarehouses(List<String> warehouses);
    void showDialog(String msg);
}

File: app/src/main/java/com/artmurka/artmurkaapp/android/views/fragments/interfaces/ICheckoutFragment.kt
package com.artmurka.artmurkaapp.android.views.fragments.interfaces


import com.artmurka.artmurkaapp.data.model.pojo.itemlist.checkout.OrderDesc
import com.artmurka.artmurkaapp.data.model.pojo.itemlist.novaposhta.Areas.AreasResponse
import com.artmurka.artmurkaapp.presenter.PresenterView

import java.util.ArrayList

interface ICheckoutFragment  : PresenterView {
    fun showCheckout(list: List<OrderDesc>)
    fun refreshSumPrice(price: String)
    fun showOrderIsProcessed(msg: String)
    fun setDataSpinner(delivery: ArrayList<String>, payment: ArrayList<String>)
    fun setSityes(sityes: Array<String>)
    fun showMessage(message: String)
    fun setAreas(areas: AreasResponse)
    fun setCities(cities: List<String>)
    fun setWarehouses(warehouses: List<String>)
    fun showDialog(msg: String)
}
